src/oauth2-server/src/Model/Casts/GrantsCast.php

```
@@ -0,0 +1,42 @@
+ * @contact  huangdijia@gmail.com
+ */
+
+namespace FriendsOfHyperf\Oauth2\Server\Model\Casts;
+
+use FriendsOfHyperf\Oauth2\Server\ValueObject\Grant;
+use Hyperf\Codec\Json;
+use Hyperf\Contract\CastsAttributes;
+
+class GrantsCast implements CastsAttributes
+{
+    public function get($model, string $key, $value, array $attributes)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$model'. (Unused Code Rules)

(UnusedFormalParameter)